lua/telescope/previewers.lua

```diff
@@ -648,7 +648,7 @@ previewers.ctags = defaulter(function(_)
         scode = string.gsub(scode, [[\/]], [[/]])
         scode = string.gsub(scode, '[*]', [[\*]])
 
-        file_maker_sync(entry.filename, self.state.bufnr, self.state.bufname)
+        file_maker_sync(entry.resolve_filename(entry.filename), self.state.bufnr, self.state.bufname)
```

Comment:
This needs also be updated at line 641. So the cache works and we only load a file max once and all further file accesses are just reusing the already loaded buffer.

Reply:
Aha. Cool. Fixed.